feat(perf-issues): Add table to root performance issue docs (#6152)
getsentry/sentry-docs: src/docs/product/issues/issue-details/performance-issues/index.mdx

The "Event Grouping Information" section provides details of how Sentry fingerprinted the event into the group. You can see which parts of the span evidence contributed to the fingerprint.

## Performance Issue Types

> The following table outlines the availble performance issues categorized by

Reviewer: ```suggestion
The following is a list of available performance issues:
```

<table>
   <tr>
       <th>Browser</th>
       <td>
          <ul>
            <li><a href="./n-one-api-calls/">*N+1 Api Calls</a></li>
          </ul>  
        </td>
    </tr>
   <tr>
       <th>Server</th>
       <td>
          <ul>
            <li><a href="./n-one-queries/">N+1 Queries</a></li>
            <li><a href="./consecutive-db-queries/">*Consecuitive DB Queries</a></li>
          </ul>  
        </td>
    </tr>
   <tr>
    <th>Mobile</th>
       <td>
          <ul>
            <li><a href="./main-thread-io/">*File IO on Main Thread</a></li>
            <li><a href="./n-one-api-calls/">*N+1 Api Calls</a></li>
          </ul>  
        </td> 
    </tr>
   <tr>
    <th>Serverless</th>
       <td>
          <ul>
            <li><a href="./n-one-queries/">N+1 Queries</a></li>
            <li><a href="./consecutive-db-queries/">*Consecuitive DB Queries</a></li>
          </ul>  
        </td> 
    </tr>
</table>
<Note>

> * indicates that a performance issue is only availble to Early Adopters.

Reviewer: ```suggestion
* Currently only available to Early Adopters.
```
